GlyssenTests/ProjectExportTests.cs

```diff
@@ -562,11 +577,11 @@ public void GetExportData_BlocksAreJoinedToStandardNonEnglishReferenceText_Outpu
 			Assert.IsTrue(data.All(d => d.Count == 11));
 			Assert.IsTrue(data.All(d => (string)d[exporter.GetColumnIndex(ExportColumn.BookId)] == "JUD"));
 			Assert.IsTrue(data.All(d => d.Count == exporter.GetColumnIndex(ExportColumn.VernacularTextLength) + 1));
-			Assert.AreEqual("Иуда", data[0][exporter.GetColumnIndex(ExportColumn.PrimaryReferenceText)]);
-			Assert.AreEqual("JUDE", data[0][exporter.GetColumnIndex(ExportColumn.SecondaryReferenceText)]);
+			Assert.AreEqual("Иуда Иуда 1", data[0][exporter.GetColumnIndex(ExportColumn.PrimaryReferenceText)]);
```

Comment:
This test case illustrates what seems to be the potential fallacy in doing this at all. With no punctuation or anything to help the voice actor, this seems quite prone to being misread. (Not necessarily a problem for the FCBH teams with an experience coach, but likely to trip up SIL teams doing self-recording.)